ted_sws/notice_eligibility/services/notice_eligibility.py

```diff
@@ -37,6 +37,17 @@ def check_package(mapping_suite: MappingSuite, notice_metadata: NormalisedMetada
     return True if not filtered_df.empty else False
 
 
+def transform_version_string_into_int(version_string: str) -> int:
+    """
+    Transforming a version string into a number. (example_version = "1.2.3")
+    :param version_string:
+    :return:
+    """
+    version_numbers = [int(x) for x in version_string.split(".")]
+    assert len(version_numbers) == 3
+    return ((version_numbers[0] * 100) + version_numbers[1]) * 100 + version_numbers[2]
```

Comment:
:))))))))))))

That would work

Reply:
If you would go into [0,1] space you could easily handle 1,2,3,4... digit version numbers. 
The same is valid for negative number space.